web/client/reducers/annotations.js

```diff
@@ -197,6 +198,8 @@ function annotations(state = {validationErrors: {}}, action) {
     }
     case FEATURES_SELECTED: {
         let newState = state;
+
```

Comment:
not a big deal, but before creating the PR check for things like this

empty lines are not needed and increase a little bit entropy in the code review
unless they are intentionally added to increase readability

Reply:
Thanks, got rid of unneeded empty lines.